code/game/gamemodes/blob/theblob.dm

```diff
@@ -219,27 +219,32 @@
 		if(lowertext(B.blob_reagent_datum.color) == lowertext(src.color)) // Goddamit why we use strings for these
 			return B.blob_reagent_datum.description
 	return "something unknown"
+
 /obj/structure/blob/normal
 	icon_state = "blob"
 	light_range = 0
 	obj_integrity = 21 //doesn't start at full health
 	max_integrity = 25
 	brute_resist = 0.25
 
-/obj/structure/blob/normal/update_icon()
-	..()
+/obj/structure/blob/normal/update_name()
+	. = ..()
 	if(obj_integrity <= 15)
-		icon_state = "blob_damaged"
 		name = "fragile blob"
+	else
+		name = "[overmind ? "blob" : "dead blob"]"
+
+/obj/structure/blob/normal/update_desc()
+	. = ..()
+	if(obj_integrity <= 15)
 		desc = "A thin lattice of slightly twitching tendrils."
+	else
+		desc = "A thick wall of [overmind ? "writhing" : "lifeless"] tendrils."
+
+/obj/structure/blob/normal/update_icon_state()
+	if(obj_integrity <= 15)
+		icon_state = "blob_damaged"
 		brute_resist = 0.5
```

Comment:
[No changes needed] Updating things like `brute_resist` inside icon update procs is bad form, but moving those to proper place is pretty out of scope of the PR. So let's leave them in for now (and maybe do a second pass later).